src/components/portfolio/PortfolioHeader.vue

```diff
@@ -0,0 +1,30 @@
+<script setup lang="ts">
+import '@components/VerticalBox.vue'
+</script>
+
+<template>
+  <div class="center-wrapper" style="margin-bottom: 1rem">
```

Comment:
Personal preference but I think that avoiding inline styles where possible is preferable in terms of maintainability. Sometimes you can't avoid it, though.

Reply:
Great point, it makes sense to not have an inline here too, since margin-bottom gets redefined for 2 center-wrappers